.editorconfig

```diff
@@ -0,0 +1,20 @@
+[{*.kt,*.kts}]
+ij_kotlin_code_style_defaults = KOTLIN_OFFICIAL
+
+max_line_length = 140
```

Comment:
Will it be enough? I think with current screens we could increase to 180

Reply:
For me, it could be 180, and if we discuss it in near future and decide it is not a good value we can change it.

Reply:
DONE

Reply:
@Nataniel-Antosik Please create an issue where everyone should post preferred length, for me 180 is too long :)

Reply:
I created issue for this [Max line length in .editorconfig#41](https://github.com/intive/Picover/issues/41)